### PR TITLE
Sync only offline runs

### PR DESCRIPTION
**Pull request by Blaizzy**

### CODE REVIEW

**Review by pkasprzyk — COMMENTED** [review]
Github doesn't allow me to comment on non-updated lines, so I'll write it here:

I think you can replace 
https://github.com/neptune-ai/neptune-client/pull/894/files#diff-4d1af3d92b849386a88c5ace8eef8d8f3c60c30caaf857de8a174692ce5e1441R257-R258 in `sync_all_containers` with call to `sync_all_offline_containers` to avoid code duplication.

Apart from that lgtm.

**Comment by Blaizzy** [comment]
Thank you  @pkasprzyk!

It is done ✅

**Review by pkasprzyk — APPROVED** [review]

**Comment by pkasprzyk** [comment]
@Blaizzy one more thing: could you add changelog entry?

**Comment by Blaizzy** [comment]
Sure I'll do it 👍🏽

**Comment by Blaizzy** [comment]
@pkasprzyk I've updated the changelog for the next release. 
Please let me know if there are any other improvements.

**Comment by Blaizzy** [comment]
@pkasprzyk I've updated the changelog for the next release. 
Please let me know if there are any other improvements.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+## neptune-client 0.16.2
```
Comment (pkasprzyk) [inline CHANGELOG.md:1]: One more thing (I haven't mentioned that before, sorry):
When you create a new version entry in CHANGELOG, please mark it with `[UNRELEASED]`, so everyone can tell at a glance that they can add new entries to it. Admittedly we don't ALWAYS do that, but at least try ;). Example: https://github.com/neptune-ai/neptune-client/pull/825/files

**Comment by Blaizzy** [comment]
Thanks!

I have made the changes ✅

**Review by pkasprzyk — APPROVED** [review]

**Pull request merged**